README.md

```diff
@@ -1,37 +1,29 @@
-# skeleton
+# @fastify/one-line-logger
 
-Template repository to create standardized Fastify plugins.
+`@fastify/one-line-logger` helps you format fastify's log into a nice one line message:
 
-# Getting started
+```
+YYYY-MM-dd HH:mm:ss.SSS - <level> - <method> <route path> - <message>
+```
 
-- Click on `Use this template` above to create a new repository based on this repository.
+A standard incoming request log line like:
 
-# What's included?
+```
+{"level": 30,"time": 1660151282194,"pid": 1557,"hostname": "foo","reqId": "req-1","req": {"method": "GET","url": "/path","hostname": "localhost:8080","remoteAddress": "127.0.0.1"},"msg": "incoming request"}
+```
 
-1. Github CI Actions for installing, testing your package.
-2. Github CI Actions to validate different package managers.
-3. Dependabot V2 config to automate dependency updates.
-4. Template for the GitHub App [Stale](https://github.com/apps/stale) to mark issues as stale. 
-5. Template for the GitHub App [tests-checker](https://github.com/apps/tests-checker) to check if a PR contains tests.
+Will format to:
 
-# Repository structure
+```
+2022-08-11 01:08:02.194 - info - GET / - incoming request
+```
+
+# Getting started
 
+```js
+const server = fastify({
+  logger: {
+    transport: "one-line-logger",
```

Comment:
A note should be in this readme indicating that the module needs to be installed for this line to work correctly.